spaces/domains.json

```
@@ -442,5 +442,6 @@
   "vote.litedex.io": "litedex.eth",
   "vote.scarychaincapital.com": "scarychaincapital.eth",
   "vote.matcha.fi": "streamingfast.eth",
-  "vote.bsc-mvb.io": "mvb.eth"
+  "vote.bsc-mvb.io": "mvb.eth",
+  "gov.diversecapital.finance":"diversecapital.eth"
```

Please add a space between the subdomain and ENS.
```suggestion
  "gov.diversecapital.finance": "diversecapital.eth"
```